src/App/ResetCryptoStore.ts

```diff
@@ -0,0 +1,89 @@
+import { rm } from "fs/promises"
+
+import { BridgeConfig, parseRegistrationFile } from "../Config/Config";
+import { Logger } from "matrix-appservice-bridge";
+import { LogService, MatrixClient } from "matrix-bot-sdk";
+import { getAppservice } from "../appservice";
+import BotUsersManager from "../Managers/BotUsersManager";
+
+const log = new Logger("App");
+
+async function start() {
+    const configFile = process.argv[2] || "./config.yml";
+    const registrationFile = process.argv[3] || "./registration.yml";
+    const config = await BridgeConfig.parseConfig(configFile, process.env);
+    const registration = await parseRegistrationFile(registrationFile);
+    Logger.configure({
+        console: config.logging.level,
+        colorize: config.logging.colorize,
+        json: config.logging.json,
+        timestampFormat: config.logging.timestampFormat
+    });
+    LogService.setLogger(Logger.botSdkLogger);
+
+    const {appservice, storage, cryptoStorage} = getAppservice(config, registration);
+    if (!cryptoStorage) {
+        log.info(`Encryption is not enabled in the configuration file "${configFile}", so there is no encryption state to be reset`);
+        return;
+    }
+
+    const botUsersManager = new BotUsersManager(config, appservice);
+
+    for (const botUser of botUsersManager.botUsers) {
+        try {
+            const userStorage = storage.storageForUser?.(botUser.userId)
+            if (!userStorage) {
+                log.warn(`No storage for ${botUser.userId}`);
+                continue;
+            }
+            const accessToken = await Promise.resolve(userStorage?.readValue("accessToken"));
```

Comment:
Hm? Oh, is this a Promise<T>|T type? You should be able to await it without the Promise.resolve.

Reply:
[From MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#description):
> ```javascript
> await expression
> ```
> ...
> The expression is resolved in the same way as [Promise.resolve()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve): it's always converted to a native Promise and then awaited.

So the `Promise.resolve` here is indeed redundant.